AUTHORS.md

```diff
@@ -0,0 +1,7 @@
+# AUTHORS
+
+The Image Map module would not be possible without the generous contributions of code, testing, and ideas from the CTS-IT developers, the REDCap Team, and Mike Conlon, Ph.D. We would also like to acknowledge the generous support of Dr. Carl J Pepine's WARRIOR project.
+
+We specifically want to thank our developers Kyle Chesney kyle.chesney@ufl.edu, James Pence jrpence@ufl.edu, and Philip Chase pbc@ufl.edu for their contributions to the project.  Taryn Stoffs tls@ufl.edu and Jerri Johnston j.johnston@ufl.edu provided valuable input into the design and testing of the Image Map module.
+
+Geoffroey-Allen Franklin frgeof Provided the new image and mapping for Pirads v2.1.
```

Comment:
Jane Coe Johnson was a co-author on 3bbb2e0

Reply:
Actually, there's already a precedent for attribution in the README for people who added images:

https://github.com/ctsit/imagemap#acknowledgements--copyright

It's also missing quite a few people!
https://github.com/ctsit/imagemap/graphs/contributors